packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -1,10 +1,30 @@
 @import '../../../../../fxa-content-server/app/styles/variables';
+@import '../../../../../fxa-content-server/app/styles/breakpoints';
 
 .product-payment {
+  background: #fff;
```

Comment:
I agree we can wait on adding new photon colors to `_variables`, but we can still add ones like `$color-white` or wherever else we have colors set that are already defined (even if just to make the SCSS linter be quiet).